config.yaml

```diff
@@ -59,9 +58,13 @@ calendar:
 # Increase to run multiple years in a single submission
 runspersub: 1
 
+collate:
+    enable: false
+
 userscripts:
     # Setup land use changes
     setup: ./scripts/pre.sh work/atmosphere/INPUT/cableCMIP6_LC_1850-2015.nc
+    archive: ./scripts/archive.sh
```

Comment:
Just checking you want to do this in the same process as the model run. If the archive step takes a decent amount of time you may want to run this as a separate PBS job via `postscript`.

Reply:
Yep that's what I was thinking - these scripts don't take that long compared to running the model itself so it seemed safe to include them in the same PBS job.